src/ansys/fluent/core/solver/flobject.py

```diff
@@ -243,6 +244,32 @@ def set_state(self, state: StateT = None, **kwargs):
         else:
             return self.flproxy.set_var(self.path, self.to_scheme_keys(state))
 
+    def find_child(self, identifier=""):
+        _list_of_children = []
+        SettingsBase._list_children(self.__class__, identifier, [], _list_of_children)
+        return _list_of_children
+
+    @staticmethod
+    def _list_children(cls, identifier, path, _list_of_children):
+        if issubclass(cls, NamedObject):
+            if hasattr(cls.child_object_type, "child_names"):
+                SettingsBase._get_child_path(
+                    cls.child_object_type, path, identifier, _list_of_children
+                )
+        if issubclass(cls, Group):
+            SettingsBase._get_child_path(cls, path, identifier, _list_of_children)
+
+    @staticmethod
+    def _get_child_path(cls, path, identifier, _list_of_children):
+        for name in cls.child_names:
+            path.append(name)
+            if identifier == "" or fnmatch.fnmatch(name, identifier):
```

Comment:
I think we can remove the `identifier == ""` condition. `"*"` will return everything.

Reply:
Thank you. Done that.

Reply:
@prmukherj: please could you add a test so that I can fully understand the behaviour. Thank you.

Reply:
Sure @seanpearsonuk, doing that currently. Will commit it soon. Thank you

Reply:
Tests have been put in place.